addons/2doptics/reticles/pip.rvmat

```diff
@@ -0,0 +1,20 @@
+ambient[] = {1,1,1,1};
+diffuse[] = {1,1,1,1};
+forcedDiffuse[] = {0,0,0,0};
+emmisive[] = {1,1,1,1};
+specular[] = {1,1,1,1};
+specularPower = 1;
+PixelShaderID = "Normal";
+VertexShaderID = "Basic";
+
+class Stage1 {
+    texture = "#(argb,8,8,3)color(0.5,0.5,0.5,0.5,dt)";
+    uvSource = "tex";
+
+    class uvTransform {
+        aside[] = {1,0,0};
+        up[] = {0,1,0};
+        dir[] = {0,0,0};
```

Comment:
```suggestion
        dir[] = {0,0,1};
```

Reply:
Explain?

Reply:
It's a matrix.
3 direction vectors (side,up,dir) and position.
0,0,0 is a invalid direction vector.

Reply:
Makes sense, but why is it ZYX?

Reply:
You are asking me why arma does weird things?

Reply:
XZY sounds more "normal" (for BI standards).